### DISCUSSION TITLE
How to select Custom Field on populated class

### DISCUSSION THREAD

**Discussion by NitelPhyoe** [mikro-orm/mikro-orm#1233] [Q&A] [answered]
In my case

```ts
class A{
  name:string;
  b: B
}

class B{
  first:string
  last:string 
  wanted:string 
}
```

When populate A to B
I only wanted some of B field like that

```
{
  name
  b:{
    wanted
  }
}
```

is there any tips to do that?

Sorry for bad English

**Comment by B4nan** [comment, marked as answer]
Either use a QB or wait for #221 

Or load it in 2 steps, first load `A`, then load `B` using the `fields` option. 

Or you could mark some props of `B` as `lazy: true` so they would be ignored automatically (and you would need to populate those explicitly to see the values).
